docs/quickstart-vsphere.md

```diff
@@ -0,0 +1,294 @@
+# How To Install Kubernetes On vSphere Cluster Using KubeOne
+
+In this quick start we're going to show how to get started with KubeOne on
+vSphere. We'll cover how to create the needed infrastructure using our example
+Terraform scripts and then install Kubernetes. Finally, we're going to show how
+to destroy the cluster along with the infrastructure.
+
+As a result, you'll get Kubernetes 1.14.1 High-Available (HA) clusters with
+three control plane nodes and two worker nodes.
+
+### Prerequisites
+
+To follow this quick start, you'll need:
+
+* `kubeone` v0.7.0 or newer installed, which can be done by following the
```

Comment:
Actually, you need `master` or `v0.8.0`, because `v0.7.0` doesn't support vSphere.

Reply:
👍 done